plugins/GoogleSearchConsole/v1/docs/README.md

```
@@ -0,0 +1,95 @@
+
+## Creating OAuth 2.0 Credentials
+
+1. From the [Google Cloud Console](https://console.cloud.google.com/), navigate to **APIs & Services** > **Credentials**
+
+2. Click **+ Create credentials** and select **OAuth client ID**
+
+3. If prompted, configure the OAuth consent screen:
+    - Choose **External**
+    - Enter the required application details
+    - Add the following scope:
+      ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)